### PR TITLE
TIP-878: Move UI bundle to platform

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | yes
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -563,3 +563,4 @@
 - Move `Pim\Bundle\DashboardBundle\Widget\LastOperationsWidget` to `Akeneo\Platform\Bundle\DashboardBundle\Widget\LastOperationsWidget`
 - Move `Pim\Bundle\DashboardBundle\Widget\Registry` to `Akeneo\Platform\Bundle\DashboardBundle\Widget\Registry`
 - Move `Pim\Bundle\DashboardBundle\Widget\WidgetInterface` to `Akeneo\Platform\Bundle\DashboardBundle\Widget\WidgetInterface`
+- Move `Pim\Bundle\UIBundle` to `Akeneo\Platform\Bundle\UIBundle\Twig`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: remove Twig?

File: UPGRADE.md [hunk 2]
```diff
@@ -528,3 +528,4 @@ find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\DashboardBundle\\P
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\DashboardBundle\\Widget\\LastOperationsWidget/Akeneo\\Platform\\Bundle\\DashboardBundle\\Widget\\LastOperationsWidget/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\DashboardBundle\\Widget\\Registry/Akeneo\\Platform\\Bundle\\DashboardBundle\\Widget\\Registry/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\DashboardBundle\\Widget\\WidgetInterface/Akeneo\\Platform\\Bundle\\DashboardBundle\\Widget\\WidgetInterface/g'
+find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\Bundle\UIBundle/Akeneo\Platform\Bundle\UIBundle\Twig/g'
```
Comment (jjanvier) [inline UPGRADE.md]: remove Twig?

File: app/AppKernel.php [hunk 3]
```diff
@@ -95,7 +95,7 @@ protected function getPimBundles()
             new Pim\Bundle\LocalizationBundle\PimLocalizationBundle(),
             new Akeneo\Platform\Bundle\NotificationBundle\PimNotificationBundle(),
             new Pim\Bundle\ReferenceDataBundle\PimReferenceDataBundle(),
-            new Pim\Bundle\UIBundle\PimUIBundle(),
+            new Akeneo\Platform\Bundle\UIBundle\PimUIBundle(),
```
Comment (jjanvier) [inline app/AppKernel.php]: should we call it PimUIBundle or AkeneoUIBundle?

Comment (arnolanglade) [reply app/AppKernel.php]: dunno I just moved the bundle without thinking about the naming.

File: UPGRADE.md (hunk 2 above)
Comment (arnolanglade) [reply UPGRADE.md]: you are right

**Comment by arnolanglade** [comment]
2 behat scenarios failed on the CI but they pass locally.

**Pull request merged**